govcd/vdc.go

```diff
@@ -560,3 +560,22 @@ func (vdc *Vdc) FindMediaImage(mediaName string) (MediaItem, error) {
 	util.Logger.Printf("[TRACE] Found media record by name: %#v \n", mediaResults)
 	return *newMediaItem, nil
 }
+
+// GetMetadata() function returns meta data for vDC.
+func (vdc *Vdc) GetMetadata() (*types.Metadata, error) {
+	return getMetadata(vdc.client, getAdminVdcURL(vdc.Vdc.HREF))
+}
+
+// DeleteMetadata() function deletes metadata by key provided as input
+func (vdc *Vdc) DeleteMetadata(key string) (Task, error) {
+	return deleteMetadata(vdc.client, key, getAdminVdcURL(vdc.Vdc.HREF))
+}
+
+// AddMetadata() function adds metadata key, value pair provided as input to vDC.
+func (vdc *Vdc) AddMetadata(key string, value string) (Task, error) {
+	return addMetadata(vdc.client, key, value, getAdminVdcURL(vdc.Vdc.HREF))
```

Comment:
Is this GitHub portal or bright sun, but I can't find actual implementation of these `addMetadata` and other functions in this PR. Help :)

Reply:
This using function already existing in code base